pkgs/data/fonts/cabin/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "cabin-1.005";
+
+  src = fetchFromGitHub {
+    owner = "impallari";
+    repo = "Cabin";
+    rev = "982839c790e9dc57c343972aa34c51ed3b3677fd";
+    name = "${name}-src";
```

Comment:
The name attribute here is not really necessary. It's pretty much only useful for derivations with multiple sources, such as the Caslon font.